sealtk/gui/AbstractItemRepresentation.cpp

```diff
@@ -209,9 +209,11 @@ QVariant AbstractItemRepresentation::data(
 
         case core::NameRole:
         case core::ClassificationTypeRole:
-        case core::ClassificationScoreRole:
           return sm->data(sourceIndex, dataRole);
 
+        case core::ClassificationScoreRole:
+          return QString::number(sm->data(sourceIndex, dataRole).toDouble(), 'f', 5);
```

Comment:
This line is too long; please wrap somehow

Reply:
Done